lang/cpp/LC_DS.h

```diff
@@ -0,0 +1,14 @@
+struct ListNode {
```

Comment:
Should we add the `ifndef define` header guard here? Maybe it's unnecessary because those `.h` files are never "included" in a cpp source file, but rather directly plugged in.

Reply:
I don't have an opinion, both are okay to me.

Reply:
Let's leave it as is then, to avoid clutter in the generated .cpp file.